.github/pull_request_template.md

```diff
@@ -3,8 +3,7 @@ Thanks for opening a PR to scanpy!
 Please be sure to follow the guidelines in our contribution guide (https://scanpy.readthedocs.io/en/latest/dev/index.html) to familiarize yourself with our workflow and speed up review.
 -->
 
-<!-- Please check (“- [x]”) and fill in the following two boxes -->
+<!-- Please check (“- [x]”) and fill in the following box -->
 - [ ] Closes #
-- [ ] Tests included or not required because:
```

Comment:
I would like to keep this line (because we often have to ask people for tests), but just don't think we actually need to check it. It would be fine if this line was deleted for some PRs.